js/iframeResizer.js

```diff
@@ -279,7 +297,19 @@
 
 			debouce(debouncedTrigger,32);
 		}
+		
+		function sendPageLocationToIframe(iframe,iframeId){
+			function debouncedTrigger(){
+				trigger(
+					'Send Page Location',
+					'pageLocation:' + getPageLocation(), 
+					iframe, 
+					iframeId
+				);
+			}
 
+			debouce(debouncedTrigger,32);
+		}
```

Comment:
Please check the other pull-request too.
I did this because i need to get the window.location from server side to know if  the server is enabled to display my site.
Please, look "iframeResizer.contentWindow.js" modifications.
:smile: